integration_tests/test_const_list.py

```diff
@@ -0,0 +1,19 @@
+from lpython import i32, Const
+
+
+def test_const_list():
+    CONST_INTEGER_LIST: Const[list[i32]] = [1, 2, 3, 4, 5, 1]
+
+    print(CONST_INTEGER_LIST.count(1))
+    print(CONST_INTEGER_LIST.index(1))
+    assert CONST_INTEGER_LIST.count(1) == 2
+    assert CONST_INTEGER_LIST.index(1) == 0
```

Comment:
I would advice adding print statements before the asserts as above. This helps during debugging later.